SubProject.cmake

```diff
@@ -241,4 +272,11 @@ if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/.gitsubprojects")
     endif()
     add_dependencies(update ${PROJECT_NAME}-update-git-subprojects)
   endif()
+
+  if(NOT ${name}_IS_SUBPROJECT)
```

Comment:
Should be ${PROJECT_NAME}_IS_SUBPROJECT. Otherwise you actually rely on ${name} being undefined...